pkg/server/backend_manager.go

```diff
@@ -53,8 +54,12 @@ func (b *backend) Context() context.Context {
 	return b.conn.Context()
 }
 
-func newBackend(conn agent.AgentService_ConnectServer) *backend {
-	return &backend{conn: conn}
+func (b *backend) AgentID() string {
```

Comment:
See https://github.com/kubernetes-sigs/apiserver-network-proxy/pull/125/files#r460320297. We might need to expose the `conn`, not the `agentID`

Reply:
ack, I'll revise this with the conn instead.